templates/aws-tap-entrypoint-new-vpc.template.yaml

```diff
@@ -639,7 +604,7 @@ Resources:
         TanzuNetPassword: !Ref TanzuNetPassword
         TanzuNetApiToken: !Ref TanzuNetApiToken
         TanzuNetRelocateImages: !Ref TanzuNetRelocateImages
-        SampleAppName: !Ref SampleAppName
+        SampleAppName: tanzu-java-web-app-workload
```

Comment:
*suggestion/non-blocking*:
Similar as [with the ssh/AMI parameters](https://github.com/vmware-tap-on-public-cloud/quickstart-vmware-tanzu-application-platform/pull/75/files#r1121554217), I think we should rip everything out of the "existing vpc" template / stack

Reply:
This [seems to fail the pipeline now](https://runway-ci-sfo.eng.vmware.com/teams/tappc/pipelines/qs-test/jobs/pr-test-multi-us-west-2/builds/93#L63f6d900:19), because we've removed that param from the "existing-vpc" template; thus we must not pass that in here.